test_settings.py

```diff
@@ -115,6 +115,7 @@ def root(*args):
 
 # URL for the server that django client listens to by default.
 TEST_SERVER = "http://testserver"
+ALLOWED_HOSTS = ["testserver.enterprise"]
```

Comment:
> Changed in Django 1.11:
> In older versions, ALLOWED_HOSTS wasn’t checked when running tests.
> In older versions, ALLOWED_HOSTS wasn’t checked if DEBUG=True. This was also changed in Django 1.10.3, 1.9.11, and 1.8.16 to prevent a DNS rebinding attack.